set.mm

```diff
@@ -18340,6 +18331,14 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
   sbid $p |- ( [ x / x ] ph <-> ph ) $=
     ( weq wsb wb equid sbequ12r ax-mp ) BBCABBDAEBFABBGH $.
 
+  ${
+    $d t x $.  $d x ph $.
+    $( Reverting substitution yields the original expression.  (Contributed by
+       Wolf Lammen, 5-Aug-2023.) $)
+    sbid2vv $p |- ( [ t / x ] [ x / t ] ph <-> ph ) $=
```

Comment:
I think it should be labeled sbid2vw (like equsalvw, etc.). We should probably review these labelings in the future, but let's be consistent and for the moment stick with the existing pattern. I would also add a link to sbid2v in the comment, e.g., the usual: "Version of ~sbid2v with a DV condition requiring fewer axioms."